specification/datasets/cost_and_usage/providercalculatedsplitcostallocation.md

```diff
@@ -0,0 +1,27 @@
+# Provider-Calculated Split Cost Allocation
```

Comment:
nit: Given the convos around different types of "provider", I'm not sure we should use the word provider here.